packages/realm/scripts/submit-analytics.mjs

```diff
@@ -330,6 +328,8 @@ async function submitAnalytics() {
     return;
   }
 
+  saveBundleId(data["Anonymized Bundle Id"]);
```

Comment:
I think we need to save the bundle id before line 326 as it is needed at runtime too.

Reply:
If this line doesn't execute on the end-users machine, it will fallback to the value already stored in the `package.json`: https://github.com/realm/realm-js/blob/main/packages/realm/package.json#L34